borgmatic/config/validate.py

```diff
@@ -137,9 +137,17 @@ def normalize_repository_path(repository):
 
 def repositories_match(first, second):
     '''
-    Given two repository paths (relative and/or absolute), return whether they match.
-    '''
-    return normalize_repository_path(first) == normalize_repository_path(second)
+    Given two repository dicts with keys 'path' (relative and/or absolute),
+    and 'label', return whether they match.
```

Comment:
Maybe this should say: "Given two repository dicts or path strings ..."? Or something similar to suggest that it accepts either one.